app/views/user_mailer/in_person_verified.html.erb

```diff
@@ -0,0 +1,60 @@
+<%= image_tag(
+      'email/user-signup-ial2.png',
+      width: 140,
+      height: 177,
+      alt: '',
+      class: 'float-center padding-bottom-4',
+    ) %>
+<h4><%= message.subject %></h4>
+<p>
+<div class="s16">
```

Comment:
Similar comment about merging tags.

```suggestion
<p class="s16">
```

Reply:
I made this change in e87a1979eeb6f903b2d66225e73508dc6b6c5654 (in #6676, which will merge into this PR)